### PR TITLE
Pyne source

### PR DESCRIPTION
**Pull request by shimwell**
There is now an MCNP6 source.F90 which was converted from the MCNP5 source.F90

Also there are some CMakeLists.txt for both mcnp5 and mcnp6 which will allow these source.F90 files to be included in the build.

### CODE REVIEW

**Comment by ljacobson64** [comment]
It looks like you have reverted some of the cmake changes from #555

**Comment by gonuke** [comment]
Or rather, branched from old code?

**Comment by shimwell** [comment]
sorry about that, I shall try again on Monday.

**Comment by shimwell** [comment]
:-)

**Comment by makeclean** [comment]
Can we merge this?

**Comment by gonuke** [comment]
Should this really be merged to `pyne-source`? Or to `develop`?

File: src/mcnp/CMakeLists.txt [hunk 1]
```diff
@@ -7,6 +7,26 @@ elseif (CMAKE_Fortran_COMPILER_ID STREQUAL "GNU")
...
+set(LINK_LIBS_STATIC)
+list(APPEND LINK_LIBS_STATIC ${DAG_LINK_LIBS})
+list(APPEND LINK_LIBS_STATIC dagmc-static)
+list(APPEND LINK_LIBS_STATIC dagtally-static)
+if (${CMAKE_SYSTEM_NAME} STREQUAL Darwin AND ${CMAKE_CXX_COMPILER_ID} STREQUAL Clang)
+  list(APPEND LINK_LIBS_STATIC c++)
+else ()
+  list(APPEND LINK_LIBS_STATIC stdc++)
+endif (${CMAKE_SYSTEM_NAME} STREQUAL Darwin AND ${CMAKE_CXX_COMPILER_ID} STREQUAL Clang)
+
```
Comment (ljacobson64) [inline src/mcnp/CMakeLists.txt]: What are all these additions? This looks like some old stuff that was removed in a previous PR

**Comment by makeclean** [comment]
@gonuke, yes pyne-src @shimwell can you update the target

File: src/mcnp/CMakeLists.txt (hunk 1 above)
Comment (makeclean) [reply src/mcnp/CMakeLists.txt]: Anything specific or all of it? To be honest there is a whole lot of churn going on so it's hard to know.

**Comment by shimwell** [comment]
Sorry for that I changed to target when it was already on pyne-source.  So I have put it back now

File: src/mcnp/CMakeLists.txt (hunk 1 above)
Comment (shimwell) [reply src/mcnp/CMakeLists.txt]: I think that #555 changed the mcnp/CMakeLists.txt and removed all of this

Comment (shimwell) [reply src/mcnp/CMakeLists.txt]: I have reverted the CMakeLists.txt to the same contents as #555

**Comment by gonuke** [comment]
Sorry for all the confusion @Shimwell.  I think the big changes on #555 make it hard for the rest of us to appreciate what the purpose of this PR is, and also which changes remain after those changes in #555 .  I think #555 may have had some overlapping and common purpose with your changes.

**Comment by ljacobson64** [comment]
I am going to recommend that this PR wait until #585 is merged, and then have this PR merge into `develop` instead.

**Comment by gonuke** [comment]
@Shimwell - we have now merged #585 so hopefully this provides a more stable place for your to make your change.  It might be as simple as rebasing your branch and resubmitting with a different base branch.

**Comment by gonuke** [comment]
Should all of this live in PyNE?  Currently we share the modified `Source.F90` in PyNE.  This is in part because it is theoretically usable by those who aren't doing DAGMC-based analysis (although they will need MOAB).

**Comment by ljacobson64** [comment]
This was implemented in #604

**Pull request closed without merging**